app/assets/javascripts/spree/frontend/solidus_paypal_commerce_platform/button_actions.js

```diff
@@ -11,6 +11,56 @@ SolidusPaypalCommercePlatform.postOrder = function(payment_method_id) {
   })
 }
 
+SolidusPaypalCommercePlatform.createAndSendOrder = function(payment_method_id) {
+  return SolidusPaypalCommercePlatform.createOrder().then(function(){
+    return SolidusPaypalCommercePlatform.sendOrder(payment_method_id)
+  })
+}
+
+SolidusPaypalCommercePlatform.createOrder = function() {
+  var data = {
+    order: {
+      line_items_attributes: [{
+        variant_id: SolidusPaypalCommercePlatform.getVariantId(),
+        quantity: SolidusPaypalCommercePlatform.quantity_field.value
```

Comment:
Just for parity (but also to allow customization, I think would be better to have a `.getQuantity()` instead.

Reply:
Agreed, thanks!